crossfit/data/array/conversion.py

```diff
@@ -105,7 +105,13 @@ def __call__(self, input: Any, to: Type[ToType]) -> ToType:
         except Exception:
             pass
 
-        # TODO: Check step here
+        # 4. Try to convert directly to torch tensor
+        try:
+            import torch
+
+            return torch.tensor(np.stack(input.to_arrow().to_pylist()), device="cuda")
+        except Exception:
+            pass
```

Comment:
@VibhuJawa this fix passes the GPU CI.

As an alternative, I tried: https://github.com/rapidsai/crossfit/pull/120/commits/3bd4d6d0d2f966bb0f3823f830a1918470a82626 but it did not fix the errors.